docs/SIL.rst

```diff
@@ -6230,6 +6374,55 @@ destination (if it returns with ``throw``).
 
 The rules on generic substitutions are identical to those of ``apply``.
 
+await_async_continuation
+````````````````````````
+
+::
+
+  sil-terminator ::= 'await_async_continuation' sil-value
+                        ',' 'resume' sil-identifier
+                        (',' 'error' sil-identifier)?
+
+  await_async_continuation %0 : $UnsafeContinuation<T>, resume bb1
+  await_async_continuation %0 : $UnsafeThrowingContinuation<T>, resume bb1, error bb2
+
+  bb1(%1 : $T):
```

Comment:
What is the ownership of T here? Owned? Can I have something that is guaranteed? It would be good to call this out explicitly.

Reply:
It does always need to be owned, yeah.

Reply:
I said in the text: " The value of the ``resume`` argument is owned by the current function." Is there a way I can make it clearer?

Reply:
Mark it as @owned like it is in ossa.

Reply:
```
bb1(%1 : @owned $T)
```